@navikt/core/react/src/util/Slot.tsx

```diff
@@ -0,0 +1,85 @@
+// https://github.com/radix-ui/primitives/blob/main/packages/react/slot/src/Slot.tsx
+import * as React from "react";
+import mergeRefs from "./mergeRefs";
+
+interface SlotProps extends React.HTMLAttributes<HTMLElement> {
+  children?: React.ReactNode;
+}
+
+export const Slot = React.forwardRef<HTMLElement, SlotProps>(
+  (props, forwardedRef) => {
+    const { children, ...slotProps } = props;
+
+    return (
+      <SlotClone {...slotProps} ref={forwardedRef}>
+        {children}
+      </SlotClone>
+    );
+  }
+);
+
+interface SlotCloneProps {
+  children: React.ReactNode;
+}
+
+const SlotClone = React.forwardRef<any, SlotCloneProps>(
+  (props, forwardedRef) => {
+    const { children, ...slotProps } = props;
+
+    if (React.isValidElement(children)) {
+      return React.cloneElement<any>(children, {
+        ...mergeProps(slotProps, children.props),
+        ref: forwardedRef
+          ? mergeRefs<any>([forwardedRef, (children as any).ref])
```

Comment:
Ser ut som det går bra uten `<any>` på `mergeRefs`
```suggestion
          ? mergeRefs([forwardedRef, (children as any).ref])
```

Reply:
Fikset 👍